src/platform/forms/tests/save-in-progress/ApplicationStatus.unit.spec.jsx

```diff
@@ -142,11 +137,7 @@ describe('schemaform <ApplicationStatus>', () => {
         formConfig={formConfigDefaultData}
       />,
     );
-
     expect(tree.subTree('.usa-alert-info')).to.not.be.false;
-    expect(tree.subTree('.usa-button-primary').text()).to.equal(
-      'Continue your application',
-    );
```

Comment:
Why did these assertions get removed? If they're getting the default text, I would expect these to still pass.

Reply:
I had removed them as they were failing but it was because I was passing in empty strings for the constants. I just updated the default data for the test and added them back.